python_modules/libraries/dagster-dg/dagster_dg/utils/plus/gql.py

```diff
@@ -7,3 +7,19 @@
     }
 }
 """
+
+LOCAL_SECRETS_FILE_QUERY = """
+{
+	secretsOrError {
+    __typename
+    ... on Secrets {
```

Comment:
can we add the OrError handling part here? particularly in the 'you are not authorized to fetch' case i think it will be useful